BackofficeBundle/Resources/public/smartadmin/js/app.js

```diff
@@ -762,6 +762,11 @@ $.fn.extend({
 				} // end else
 			} // end if
 		});
+
+		// prevent url load on extends accordion menu by click on collapse sign

```

Comment:
This file should not be modified as it is part of an external component. Is it possible to put this code in an open-orchestra script file ?